### PR TITLE
Improve variables documentation

### LINKED ISSUE #14143: Add support for MariaDB documentation
### Steps to reproduce
1. Run recent MariaDB
2. Visit server variables page
3. Open documentation link

### Expected behaviour
The MariaDB documentation should open

### Actual behaviour
The MySQL documentation opens

### Related code
https://github.com/phpmyadmin/phpmyadmin/blob/695fe3a994b18ad21a4c1119305e25edd71c7036/libraries/classes/Util.php#L318-L355

### PR DESCRIPTION
**Pull request by williamdes**
Partially fixes : #14143 

From what I tested almost all existing variables have documentation when using this code.

- Uses preferential mode MySQL docs for MySQL users and MariaDB docs for MariaDB users
- Unit tests now use data that only exists for tests
- Added package williamdes/mariadb-mysql-kbs

Original commits :
- https://github.com/williamdes/phpmyadmintest/tree/docsv1
- https://github.com/williamdes/phpmyadmintest/commit/2d6cc8d2c7285dbe45add9f2124ba90a4f8b0dcc
- https://github.com/williamdes/phpmyadmintest/commit/c80e1b2c0f5e3787c2ac90f7d61a706948bbc95a
- https://github.com/williamdes/phpmyadmintest/commit/5753281fa33e056968f8a4d7a62c9db854774b97
- https://github.com/williamdes/phpmyadmintest/commit/6ebf7739d9758d558ded996cb32ee843b20c6525

### CODE REVIEW

**Comment by williamdes** [comment]
@ibennetch Can you have a look on this PR please ?

**Comment by williamdes** [comment]
@nijel Can you please have a look ?

**Review by ibennetch — APPROVED** [review]
I think this is an excellent improvement. It looks good to me and the functionality improvement for users is going to be great.

**Comment by williamdes** [comment]
@nijel @mauriciofauth ?

File: libraries/classes/Util.php [hunk 1]
```diff
@@ -356,34 +358,76 @@ public static function getMySQLDocuURL($link, $anchor = '')
...
-                . self::getImage('b_sqlhelp', __('Documentation')) . '</a>';
+        bool $bigIcon = false,
+        $url = null,
+        $text = null,
+        $anchor = ''
+    ): string {
+        if ($url === null) {
+            $url = self::getMySQLDocuURL($link, $anchor);
+        }
+        $openLink = '<a href="' . $url . '" target="mysql_doc">';
```
Comment (nijel) [inline libraries/classes/Util.php]: Maybe htmlspecialchars should be used here around $url?

Comment (williamdes) [reply libraries/classes/Util.php]: Done !

**Comment by williamdes** [comment]
@nijel I fixed what you suggested, PHPUnit tests are broken because of something in the Romanian translation, I can not fix this.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**